docs/backends/macos.rst

```diff
@@ -28,7 +28,31 @@ As stated above, this will however only work the macOS machine that performed
 the scan and thus cached the device as ``243E23AE-4A99-406C-B317-18F1BD7B4CBE``.
 
 There is also no pairing functionality implemented in macOS right now, since it does not seem
-to be any explicit pairing methods in the COre Bluetooth.
+to be any explicit pairing methods in the Core Bluetooth.
+
```

Comment:
Would be nice to add a section heading here. (probably have add another one above for the existing section too for it to make sense)

Reply:
Sections added!